runtime/compiler/env/j9methodServer.cpp

```diff
@@ -105,8 +105,7 @@ TR_ResolvedJ9JITaaSServerMethod::isJNINative()
    {
    if (supportsFastJNI(_fe))
       return _jniTargetAddress != NULL;
-   _stream->write(JITaaS::J9ServerMessageType::ResolvedMethod_isJNINative, _remoteMirror);
-   return std::get<0>(_stream->read<bool>());
+   return _isJNINative;
```

Comment:
If we already cache the right result, do we need the `supportsFastJNI` check above?

Reply:
The supportsFastJNI is for TR_TARGET_S390 TR_TARGET_X86 and TR_TARGET_POWER. If it is for any of these target then  _jniTargetAddress is used to make decision and for other target  isJNINative is used.